src/app/lists/ListDataview.tsx

```diff
@@ -203,15 +206,32 @@ const ListsDataView: React.FC<ListDataViewProps> = ({
                 .delete()
                 .eq("primary_key", itemToDelete.primaryKey);
 
+            const auditLog = {
+                action: "delete a list item",
+                content: {
+                    itemName: itemToDelete.itemName,
+                    itemPrimaryKey: itemToDelete.primaryKey,
+                },
+            } as const satisfies Partial<AuditLog>;
+
             if (error) {
                 const logId = await logErrorReturnLogId(
                     `Error with delete: Ingredient id ${itemToDelete.primaryKey}`,
-                    error
+                    { error: error }
                 );
-                setErrorMsg(error.message + `Error ID: ${logId}`);
-            } else {
-                window.location.reload();
+                await sendAuditLog({
+                    ...auditLog,
+                    wasSuccess: false,
+                    logId,
+                    listId: itemToDelete.primaryKey,
+                });
+                handleSetError(error.message + `Error ID: ${logId}`);
```

Comment:
We probably won't want to rely on the error message coming out of Supabase here because we don't know ho
```suggestion
                handleSetError(`Failed to delete a list item. Log ID: ${logId}`);
```